lms/templates/admin/base.html.jinja2

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>{% block title %}LMS Admin{% endblock %}</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.2/css/bulma.min.css">
```

Comment:
I googled a bit and this one didn't seem to assume you were going to use some java script and just makes moving things around easier.

Reply:
Can we host this file ourselves (in `lms/static/styles/`) instead of linking to `jsdelivr.net`?